src/main/java/gumtree/spoon/builder/SpoonGumTreeBuilder.java

```diff
@@ -35,13 +37,12 @@
 
 	private final TreeContext treeContext = new TreeContext();
 
-	public ITree getTree(CtElement element) {
-		final ITree root = treeContext.createTree(-1, "", "root");
+	public Tree getTree(CtElement element) {
+		Type type = type("root");
+		final Tree root = treeContext.createTree(type, "");
 		new TreeScanner(treeContext, root).scan(element);
 
-		root.refresh();
-		TreeUtils.postOrderNumbering(root);
-		TreeUtils.computeHeight(root);
+		// root.refresh();
```

Comment:
If this is not required, maybe we can remove this?

Reply:
I will remove that commented code.